packages/features/bookings/lib/getLocationOptionsForSelect.ts

```diff
@@ -18,11 +19,11 @@ export default function getLocationsOptionsForSelect(
         return null;
       }
       const type = eventLocation.type;
+      const translatedLocation = getTranslatedLocation(location, eventLocation, t);
 
       return {
         // XYZ: is considered a namespace in i18next https://www.i18next.com/principles/namespaces and thus it get's cleaned up.
-        // Beacause there can be a URL in here, simply don't translate it if it starts with http: or https:. This would allow us to keep supporting namespaces if we plan to use them
-        label: locationString.search(/^https?:/) !== -1 ? locationString : t(locationString),
+        label: translatedLocation || "",
```

Comment:
Should we have:
```
label: translatedLocation || locationString
```

Because if translation returns null, then it's better we display original one to at least know what is going on, because now simply rendering empty "" makes it easier to understand where did the original "locationString".